CHANGES_NEXT_RELEASE

```diff
@@ -1,3 +1,3 @@
-Upgrade Esper library from 6.1.0 to 7.1.0
+Upgrade Esper library from 6.1.0 to 8.3.0
```

Comment:
So at the end, we can upgrade to 8.3.0 without adding any extra repository? (I understand it differently from https://github.com/telefonicaid/perseo-core/issues/136)

Reply:
Not really: https://travis-ci.org/telefonicaid/perseo-core/builds/595955266?utm_source=github_status&utm_medium=notification